device/transport/amqp/src/amqp_device_method_client.ts

```diff
@@ -108,13 +106,16 @@ export class AmqpDeviceMethodClient extends EventEmitter {
                         this._receiverLink.on('message', (msg) => {
                           debug('got method request');
                           debug(JSON.stringify(msg, null, 2));
-                          const methodName = msg.applicationProperties[methodMessagePropertyKeys.methodName];
+                          const methodName = msg.application_properties[methodMessagePropertyKeys.methodName];
+                          //
+                          // SERIALIZATION DEPENDENCY!!
+                          // The rhea library will de-serialize an encoded uuid (0x98) as a 16 byte buffer.
+                          //
```

Comment:
I wonder if we should ask Gordon to separate the type/serialization stuff from the rest of the library - depending on rhea for both packages might create issues (diamond-dependency code smell)